gcp-observability/src/main/java/io/grpc/gcp/observability/GlobalLoggingTags.java

```diff
@@ -139,10 +132,6 @@ private static String readFileContents(String file) {
     return null;
   }
 
-  private static void populateFromEnvironmentVars(ImmutableMap.Builder<String, String> customTags) {
-    populateFromMap(System.getenv(), customTags);
-  }
-
   @VisibleForTesting
   static void populateFromMap(Map<String, String> map,
```

Comment:
Since we are no longer taking `custom_tags` from environment variables, do we still need `populateFromMap()`?

Reply:
good point. Removed unused code